src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER_OF_BALL_COUNT = 100;
+    private static final Random RANDOM = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        int ballNumber = RANDOM.nextInt(MAX_NUMBER_OF_BALL_COUNT);
+        Color ballColor = colorSupplier.getRandomColor();
```

Comment:
according to my [comment](https://github.com/mate-academy/jv-lottery/pull/1080/files#r1288345986), you should pass `String` to Ball's constructor

```suggestion
        String ballColor = colorSupplier.getRandomColor().toLowerCase();
```